index.bs

```diff
@@ -136,7 +140,7 @@ Usage example {#example}
     document.addEventListener('visibilitychange', () => {
       if (document.visibilityState === 'hidden') {
         // Force any pending records to be dispatched.
-        observer.takeRecords();
+        updateCLS(observer.takeRecords());
```

Comment:
Just to confirm - did you test that this actually works?

Reply:
Yea. takeRecords() returns the list of entries. @philipwalton FYI, is this how you'd write this?

Reply:
Yeah, works for me.

Another option is to do `observer.takeRecords().forEach(updateCLS)` in the event that `updateCLS` just takes a single entry (which I personally prefer, but don't think it really matters much).